api-module-library/hubspot/api.js

```diff
@@ -822,6 +825,58 @@ class Api extends OAuth2Requester {
         return this._post(options);
     }
 
+    // ***********************   Email Templates   **************************
+
+    async getEmailTemplates(query=''){
```

Comment:
@MichaelRyanWebber Does this return an array?

Is now the time for us to change HubSpot methods to reflect the convention of `list` for array returns? (`listEmailTemplates`)

Or is that a major version release/something we should be more thoughtful of for backwards compatibility?

Reply:
It returns an object with an array (called `objects`)
I don't actually know what the conventions are, fully. I think I had been erring towards following the API's naming convention, but I like using our own, and saying list seems great.
https://linear.app/lefthook/issue/LEF-651/update-hubspot-api-module-method-names
I'm fine with doing it now/soon, but would rather leave it out of this PR